### PR TITLE
New create map

### LINKED ISSUE #2311: Conflict with buttons in the APi keys and the oAuth pages
screen shot 2015-02-18 at 12 23 42

If you are a paid user there is a conflict when scanning the page visually between the generate again and the support button.

### LINKED ISSUE #2352: Empty dataset shouldn't be in the list of datasources
there should be an option to create an empty map, what about adding to this submenu algined to the right?

screen shot 2015-02-19 at 16 09 42

### LINKED ISSUE #2414: Prepare splash screens for the new videos
- [x] Icons and colors  (@tallon).
- [x] Short descriptions for videos (@auremoser or @andrewxhill ).
- ~~Display key points within the videos (like in attached 2 - @auremoser or @andrewxhill ).~~
- [x] What to do with the videos (@saleiva and @xavijam).
- [x] Review videos (titles, names, etc - @auremoser or @andrewxhill).

### LINKED ISSUE #2457: New empty datasets don't generate a new map

### LINKED ISSUE #2627: Show Data library datasets on the create map modal window when not having connected datasets
Like we do on the datasets dashboard. When you are creating a map and you don't have any connected dataset.

### LINKED ISSUE #2765: When any dataset is selected, 'Create Map' option should be available

### LINKED ISSUE #2766: Placeholders for emtpy map dashboard rows are missing
Remember we had some in the designs with a random CTA

### PR DESCRIPTION
**Pull request by xavijam**
In short,  user will be able to:
- Create new maps from the new dashboard datasets list (using data library too, fixes #2765).
- Create new maps from the "create dialog" using data-library. Dialog will manage necessary imports + visualization creation.

This will affect in those points:
- Filter view will "store" a selected items list.
- Users will not be able to import or select data library datasets that don't fit on user quota.
- Create a map from new-create-dialog, you could user data library datasets.
- Create empty dataset will generate a new map (fixes #2352 and fixes #2457).
- Show Data library datasets on the create map modal window when not having connected datasets (fixes #2627).

Also:
- Added template videos (fixes #2414).
- Add placeholders for empty map dashboard rows (fixes #2766).
- New support banner (fixes #2311).

Previews: 

placeholder

Creating a map with a data library (need to import it) + other datasets [[video](https://dl.dropboxusercontent.com/u/931536/import.mov)].

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2362/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2389/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2393/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2408/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2422/))

File: lib/assets/javascripts/cartodb/new_dashboard/router/model.js [hunk 1]
```diff
@@ -67,6 +67,14 @@ module.exports = cdb.core.Model.extend({
     return this.get('q') || this.get('tag');
   },
 
+  isDatasets: function() {
+    return this.get('content_type') === 'datsets';
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/new_dashboard/router/model.js]: Datsets for the win.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2426/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2429/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2430/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2437/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2444/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2447/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2448/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2449/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2450/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2457/))

**Pull request merged**